attributes/network/9_account_features/2_account_kvstore.adoc

```diff
@@ -0,0 +1,25 @@
+== KvStore Account Feature Attribute (#2)
```

Comment:
Not sure how this related on Accounts. How do you see accounts working here?

Reply:
A key owner could be an account allowing finer, multi-user ACL.

I thought a little more about it, and I think we're still missing a piece to allow an account to be a key owner.

It might be a case of YAGNI. It might also come at a later stage.

WDTY?

Reply:
Discussed during a meeting. We will keep this.